Cartfile

```diff
@@ -4,3 +4,4 @@ github "neilpa/Rex" ~> 0.9.0
 
 # Utilities
 github "CocoaLumberjack/CocoaLumberjack" ~> 2.2.0
+github "Wolox/wolmo-core-ios" "master"
```

Comment:
This PR is adding Fastlane I don't see why it should introduce a dependency that has nothing to do with Fastlane. Also we should try to avoid `master` version of our dependencies. I think we should wait until we release a first version of core.